### DISCUSSION TITLE
React.Component is undefined at runtime

### DISCUSSION THREAD

**Discussion by hydroper** [vercel/turborepo#2925] [Help] [answered]
When I move the cursor over `extends React.Component` I get a tip indicating the class exists, but when I run `npm run dev` and access the site, I get `undefined` for that class. Is that a bug or what?

I've not modified `package.json`, just deleted unused files and cleaned the starter template things. Here's more information on my issue: https://stackoverflow.com/q/74671587

```tsx
import React from 'react';
import baseResolution from '@/lib/baseResolution';

export default class GameContainer
    extends React.Component<{}, {resizeListener: Function | null}>
{
    // ...
}
```

Appreciate help!

**Comment by anthonyshew** [comment, marked as answer]
Closing for staleness (and the StackOverflow question is a 404).
